### PR TITLE
Added Description and Brief Description for Editor Inspector

### PR DESCRIPTION
**Pull request by ghost**
Noticed there is no Brief Description or Description for Editor Inspector so I created one, tell me if I should make any changes or additions.

### CODE REVIEW

File: doc/classes/EditorInspector.xml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		The tab located by default in the right hand side of the editor. It is used to edit properties of the node selected.
 	</brief_description>
 	<description>
+		The inspector is a tab by default located on the right hand side of the godot editor. It is used to edit the properties of the selected node. For example, if you hand an Sprite node selected and head to the inspector. You can edit the transform of it and other properties.The inspector is essential in the game development work flow.
```
Comment (aaronfranke) [inline doc/classes/EditorInspector.xml]: "godot" should be "Godot" (capitalize it). "an Sprite" should be "a Sprite", but this sentence is confusing and needs work. Missing space after "properties." "work flow" -> "workflow".

**Review by aaronfranke — CHANGES_REQUESTED** [review]

File: doc/classes/EditorInspector.xml (hunk 1 above)
Comment (MostafaMThabet) [reply doc/classes/EditorInspector.xml]: Thanks for pointing that out, I just fixed it.

File: doc/classes/EditorInspector.xml [hunk 2]
```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="EditorInspector" inherits="ScrollContainer" version="4.0">
 	<brief_description>
+		A tool used to edit properties of the node selected.
 	</brief_description>
 	<description>
+		The inspector is a tab by default located on the right hand side of the Godot editor. It is used to edit the properties of the selected node. For example, if you hand a Sprite node selected, you will be able to edit the properties of the node, such as seting or reseting the tranform and animations. The inspector is contains the tools that are heavly needed in your game development workflow.
```
Comment (aaronfranke) [inline doc/classes/EditorInspector.xml]: "hand" -> "have", "seting" -> "setting", "reseting" -> "resetting", "tranform" -> "transform", "heavly" -> "heavily".

**Review by aaronfranke — CHANGES_REQUESTED** [review]
Please make sure that your words are spelled correctly before pushing changes.

https://www.jspell.com/checker/

**Pull request closed without merging**